### ISSUE TITLE
Please consider putting this plugin in the WordPress plugin directory

### ISSUE THREAD

**Issue by erikgeurts** [andyhqtran/Divi-100-Article-Card#23]
#### Before submitting your issue, please ensure the following:
* [ x] You have reproduced the issue in the latest version of this plugin.
* [ x] You searched Github for the issue and found no reports of the issue currently open.
* [ x] You properly formated the issue title – `Scope :: One sentence to describe the issue`.
* [ x] You have provided a detailed description of the issue.
* [ x] You have provided any/all relevant information: screenshot, screencast.
* [ x] You have provided the steps that one must take to reproduce the issue.

_Warning: Please make sure that your issue is detailed, otherwise the ticket will not be prioritized due to lack of information._

-

### Problem:
Installing the plugin by uploading the .zip file is simple enough, but updating it isn't. It would be a lot easier to do this if the plugin was in the WordPress Plugin Directory https://wordpress.org/plugins/ and if newer version were made available through there as well.

### Steps To Reproduce:

1.
2.
3.
4.

### Additional Information:

**Divi Version**:

**Child Theme**:

**Device**:

**Operating System**:

**Browser**:

**Screen Resolution**:

**Reference**:

**Comment by BrandonShutter** [comment]
https://github.com/afragen/github-updater

**Comment by andyhqtran** [comment]
At the moment, I have no plans on adding it to the WordPress directory as it'll be difficult to keep it up to date with the amount of plugins that I'll need to maintain. What @BrandonShutter mentioned is the perfect solution to keep up to date with these plugins.

**Comment by erikgeurts** [comment]
Yes, I agree, looks promising. It does require a tiny code addition to the plugin itself, which seems reasonable.

**Comment by andyhqtran** [comment]
Yeah, I'll add in support for to the GitHub updater plugin.

- [ ] Add in support for [GitHub Updater](https://github.com/afragen/github-updater) plugin
